src/schema/v2/filterArtworksConnection.ts

```diff
@@ -230,7 +232,18 @@ export const filterArtworksArgs: GraphQLFieldConfigArgumentMap = {
   },
 }
 
-export const pageableFilterArtworksArgs = pageable(filterArtworksArgs)
+const pageableFilterArtworksArgs = pageable(filterArtworksArgs)
+
+const FilterArtworksInputType = new GraphQLInputObjectType({
+  name: "FilterArtworksInput",
+  fields: pageableFilterArtworksArgs,
+})
+
+// exported for use in stitching SDL
+export const pageableFilterArtworksArgsWithInput = {
```

Comment:
The arguments for the `FilterArtworksConnection` type are thus all of those root and connection-paging fields, as well as the `input` field (which contains all of them too).